packages/phishing-controller/src/PhishingController.test.ts

```diff
@@ -69,6 +69,89 @@ describe('PhishingController', () => {
     expect(nockScope.isDone()).toBe(false);
   });
 
+  describe('maybeUpdatePhishingLists', () => {
+    it('should not be out of date immediately after maybeUpdatePhishingLists is called', async () => {
+      const clock = sinon.useFakeTimers();
```

Comment:
Nit: This test, and the others that call `maybeUpdatePhishingLists` with no `nockScope`, should include a "default" nock scope at the beginning just to prevent the test from generating console errors:

e.g.

```suggestion
      const nockScope = nock(PHISHING_CONFIG_BASE_URL)
        .get(METAMASK_CONFIG_FILE)
        .reply(200, {
          blacklist: [],
          fuzzylist: [],
          tolerance: 0,
          whitelist: [],
          version: 0,
        })
        .get(PHISHFORT_HOTLIST_FILE)
        .reply(200, []);
      const clock = sinon.useFakeTimers();
```

Reply:
I put in in beforeEach since all the tests call `maybeUpdatePhishingLists`

Reply:
Sounds good!

Nit: Generally it's better to use _setup functions_ over `beforeEach` so that you can tailor the setup steps for each individual test. For this case for example, we could use blank lists for most tests, but use the `'this-should-not-be-in-default-blacklist.com'` entry for the test that needed it.

Just something to keep in mind for the future. This is good enough for now, better than most of our tests already.